mkt/webapps/tasks.py

```diff
@@ -1145,20 +1142,38 @@ def generate_apps(hosted=0, packaged=0, privileged=0, versions=1):
     apps = generate_app_data(hosted + packaged + privileged)
     for i, (appname, cat_slug) in enumerate(apps):
         if i < privileged:
-            app = generate_packaged_app(appname, cat_slug,
-                                        num_versions=versions,
+            app = generate_packaged_app(appname, 'privileged', cat_slug,
+                                        versions=versions,
                                         permissions=['camera', 'storage'])
         elif i < (privileged + packaged):
-            app = generate_packaged_app(appname, cat_slug,
-                                        num_versions=versions)
+            app = generate_packaged_app(appname, 'web', cat_slug,
+                                        versions=versions)
         else:
             app = generate_hosted_app(appname, cat_slug)
         generate_icon(app)
         generate_preview(app)
-        generate_translations(app)
+        generate_localized_names(app.name, 2)
```

Comment:
number of languages to localize the generated app in. (This was the previous behavior; this branch just parameterizes it now)